### PR TITLE
Update managing-builders docs

### LINKED ISSUE #51: Using TBS -- Managing Builders
Production:  https://docs.pivotal.io/build-service/0-1-0/managing-builders.html

hackmd:  https://hackmd.io/ph7cDWeLRsqudafXI2Ul6w

### PR DESCRIPTION
**Pull request by tylerphelan**
Resolves #51

### CODE REVIEW

File: managing-builders.html.md.erb [hunk 1]
```diff
@@ -3,66 +3,75 @@ title: Managing Builders
 owner: Build Service Team
 ---
 
-<p class='note'><strong>Note:</strong> Only Build Service Admins can manage Cluster scoped builders.</p>
+A Custom Builder is a Tanzu Build Service resource used to manage Cloud Native Buildpack [builders](https://buildpacks.io/docs/concepts/components/builder/). There are two types of Custom Builders:
```
Comment (mgibson1121) [inline managing-builders.html.md.erb:6]: In addition to managing CNBs, they also allow for specification of a stack

File: managing-builders.html.md.erb [hunk 2]
```diff
@@ -3,66 +3,75 @@ title: Managing Builders
...
+    kp custom-builder create <name> <tag> --order <order> --stack <stack> --store <store> --namespace <namespace>
+    ```
 
-To apply a builder configuration to Build Service:
+Where:
+* `name`: The name of the builder.
+* `tag`: The registry location where the builder will be created.
+* `namespace` The kubernetes namespace for the builder (for Custom Builders only)
+* `order`: The local path to the buildpack order YAML that the builder will use.
+  For more information about listing buildpacks in groups in the order YAML, see [builder.toml](https://buildpacks.io/docs/reference/builder-config/#order-_list-required_) in the Buildpacks.io documentation.
```
Comment (mgibson1121) [inline managing-builders.html.md.erb:33]: The product files page for the v3 release of our dependencies tile will also have sample order files.  https://network.pivotal.io/products/tbs-dependencie

File: managing-builders.html.md.erb [hunk 3]
```diff
@@ -3,66 +3,75 @@ title: Managing Builders
...
 
-1. Create a builder configuration YAML file. For example:
+  Example order YAML file:
```
Comment (mgibson1121) [inline managing-builders.html.md.erb:35]: Some context into what this order file enables would be good.  Something like "this is an example of an order file that would be used by a builder designed to build Node and Java apps.

File: managing-builders.html.md.erb [hunk 4]
```diff
@@ -3,66 +3,75 @@ title: Managing Builders
...
+	- id: paketo-buildpacks/gradle
+  - group:
+	- id: paketo-buildpacks/nodejs
+  ```
 
-    <p class='note'><strong>Note:</strong> Using the <code>pb store list --order</code> can be useful for this task as the command returns buildpacks available for use in builders in correct order definition.</p>
+## <a id='patch-builder'></a> Patching a Custom Builder
 
-1. Apply the builder configuration by running:
+Custom Builders can be patched with the same arguments as `create`:
```
Comment (mgibson1121) [inline managing-builders.html.md.erb:48]: Might be good to explain that patching the custom builder will update the resource according to the specified arguments.  This also may be unnecessary if the reader gets context on what patching means from the image patch documentation.

File: managing-builders.html.md.erb (hunk 1 above)
Comment (tylerphelan) [reply managing-builders.html.md.erb]: [See](https://github.com/pivotal-cf/docs-build-service/commit/4ac8d3232c9f4deb37b99ea5dedc7af034a7d5b0#diff-54dcf468e0c728d9c6c44810fbdffc49R8)

File: managing-builders.html.md.erb (hunk 2 above)
Comment (tylerphelan) [reply managing-builders.html.md.erb:41]: G call, [see here](https://github.com/pivotal-cf/docs-build-service/commit/4ac8d3232c9f4deb37b99ea5dedc7af034a7d5b0#diff-54dcf468e0c728d9c6c44810fbdffc49R43).

File: managing-builders.html.md.erb (hunk 3 above)
Comment (tylerphelan) [reply managing-builders.html.md.erb]: G call, [see here](https://github.com/pivotal-cf/docs-build-service/commit/4ac8d3232c9f4deb37b99ea5dedc7af034a7d5b0#diff-54dcf468e0c728d9c6c44810fbdffc49R43)

File: managing-builders.html.md.erb (hunk 4 above)
Comment (tylerphelan) [reply managing-builders.html.md.erb]: Also that the arguments aren't actually the same as `create`...

See updated section:
https://github.com/pivotal-cf/docs-build-service/commit/4ac8d3232c9f4deb37b99ea5dedc7af034a7d5b0#diff-54dcf468e0c728d9c6c44810fbdffc49R54

**Comment by tylerphelan** [comment]
@mgibson1121 responded to comments

**Comment by mgibson1121** [comment]
@tylerphelan Nice, going to approve.  But one last suggestion.  Mention the aliases `cb` and `ccb` somewhere.

**Comment by tylerphelan** [comment]
Added `ccb` and `cb` aliases

**Pull request merged**